sphinx/builders/latex/constants.py

```diff
@@ -142,6 +148,8 @@
         'fontenc':     ('\\usepackage{fontspec}\n'
                         '\\defaultfontfeatures[\\rmfamily,\\sffamily,\\ttfamily]{}'),
         'fontpkg':      XELATEX_DEFAULT_FONTPKG,
+        'fvset':        '\\fvset{fontsize=\\small}',
```

Comment:
GNU FreeFont need us to use the ``\small`` because they behave a bit like Courier vs Times